### DISCUSSION TITLE
Ασκήσεις (γραμμής εντολών) που έγιναν τον Σεπτέμβρη, ισχύουν και για την εξεταστική του Φεβρουαρίου;

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1608] [Announcements]
Έχω δώσει το μάθημα τον Σεπτέμβρη, θα μπορούσα τις ασκήσεις γραμμής εντολών που έκανα τότε να τις κρατήσω για την εξεταστική που έρχεται και να ασχοληθώ με το συμμετοχικό υλικό που ήταν και ο λόγος που δεν πέρασα το μάθημα; Θα μπορούσε κάποιος να δει τις ασκήσεις γραμμής εντολών στο branch μου ([2015002](https://github.com/mariosconsta/hci/blob/ac5ea6839aab9965a5f2d38564cc7f7c4b22cd07/projects/2015002/README.md)) και να μου επιβεβαιώσει ότι όντος αυτά είναι τα ζητούμενα και ότι είναι αποδεκτές εφόσον δεν μπορώ να τις παρουσιάσω από κοντά; </b>

Παράλληλα τότε έκανα και ασκήσεις προγραμματισμού αλλά πλέον δεν τις βλέπω στα παραδοτέα άρα υποθέτω δεν πιάνουν κάτι πλέον για τον τελικό βαθμό;

**Comment by epidrome** [comment]
https://courses-ionio.github.io/help/faq/
